source/_posts/2018-07-06-release-73.markdown

```diff
@@ -17,7 +17,7 @@ In the meanwhile, we're also working hard on the new authentication system. A pr
 
 This release also includes a TON of love for the new Lovelace UI. Yes, it's still experimental but daaang, it's already so awesome that you should probably just go ahead and try it out. Thanks to all the devs who have jumped on this: [@c727], [@ciotlosm] and [@jeradM].
 
-We don't have time to go through all the changes in this release, but [@ciotlosm] has been keeping [a very detailed changelog](https://github.com/ciotlosm/docs-lovelace/blob/0.73.0/changelog.md). I'll just leave this screenshot by [@arsaboo] of the new picture-elements card and [the Lovelace configuration](https://github.com/arsaboo/homeassistant-config/blob/master/ui-lovelace.yaml#L15-L158):
+We don't have time to go through all the changes in this release, but [@ciotlosm] has been keeping [a very detailed changelog](https://www.home-assistant.io/lovelace/changelog/). I'll just leave this screenshot by [@arsaboo] of the new picture-elements card and [the Lovelace configuration](https://github.com/arsaboo/homeassistant-config/blob/master/ui-lovelace.yaml#L15-L158):
```

Comment:
Use relative urls  `lovelace/changelog/`

Reply:
`/lovelace/changelog/` surely? Otherwise it'll go looking for it in the wrong place